src/components/DetailsOverlay/ButtonRow/index.tsx

```diff
@@ -43,8 +99,60 @@ const ButtonRow: React.FC = () => {
       >
         {hasCopied ? "Copied!" : tooltipText}
       </Tooltip>
-      <Button leftGlyph={<Icon glyph="Export" />}>RAW</Button>
-      <Button leftGlyph={<Icon glyph="Export" />}>HTML</Button>
+      {jobLogsURL && (
+        <Tooltip
+          align="top"
+          justify="middle"
+          trigger={
+            <Button
+              as="a"
+              data-cy="job-logs-button"
+              href={jobLogsURL}
+              leftGlyph={<Icon glyph="Export" />}
+              target="_blank"
+            >
+              JOB LOGS
```

Comment:
[nit] I don't think we need to go caps-lock mode here
```suggestion
              Job Logs
```